### ISSUE TITLE
Support Recorder panel in Edge, which can record & replay a user flow

### ISSUE THREAD

**Issue by xhywxtt** [MicrosoftEdge/DevTools#52]
Chrome devtools has supported the Recorder panel from Chrome 97: [Recorder: Record, replay and measure user flows - Chrome Developer](https://developer.chrome.com/docs/devtools/recorder/), which can record & replay a user flow, also can export the user flow as Puppeteer script to customize the script further.

 

I wonder whether there's any plan for Edge devtools to include this Recorder panel. And is it possible to export the user flow as Playwright script?

**Comment by captainbrosset** [comment]
Thanks for your question! @zoherghadyali will be interested in this, so let me ping him here.

As you might know, Edge is built on Chromium, which Chrome uses too, and the DevTools are part of the project.
However, my understanding is that the Recorder panel itself, is not part of Chromium but instead only exists in Chrome. That means that other Chromium-based browsers (such as Edge) don't automatically have it too.

It's great knowing that you are interested in it. We certainly have a lot of interest for test automation tools and workflows, and will consider this as part of our priorities.

**Comment by TIPWARE** [comment]
Any news on this?

**Comment by captainbrosset** [comment]
No we don't have any news to share on this at the moment unfortunately. This request is definitely on our backlog, but not currently in our prioritized list.

**Comment by pieterdewachter** [comment]
Following...

**Comment by akynaston** [comment]
Following too . . .

**Comment by mtt-artis** [comment]
It seems to be good in v120

**Comment by captainbrosset** [comment]
Correct, the Recorder tool now works in Edge DevTools. I'll update this issue when I know more details about this.
